pkg/kubefedctl/join.go

```diff
@@ -243,20 +257,22 @@ func JoinCluster(hostConfig, clusterConfig *rest.Config, kubefedNamespace,
 	}
 	klog.V(2).Infof("Created %s namespace in joining cluster", kubefedNamespace)
 
-	// Create a service account and use its credentials.
-	klog.V(2).Info("Creating cluster credentials secret")
```

Comment:
any consideration to clean SA `creating` and `created` messages?

Reply:
The removed logging was duplicating logging already being done in `createAuthorizedServiceAccount. Logging in join/unjoin remains terribly inconsistent (related: #189).